validator/client/validator.go

```diff
@@ -236,6 +238,40 @@ func (v *validator) SlasherReady(ctx context.Context) error {
 	return nil
 }
 
+// ReceiveBlocks starts a gRPC client stream listener to obtain
+// blocks from the beacon node. Upon receiving a block, the service
+// broadcasts it to a feed for other usages to subscribe to.
+func (v *validator) ReceiveBlocks(ctx context.Context) {
+	ctx, span := trace.StartSpan(ctx, "validator.ReceiveBlocks")
+	defer span.End()
```

Comment:
This isn't a good candidate for a span because this routine will never end (until the process is shut down).

Reply:
Ah yes. Thank you